docker/vnc/Dockerfile

```diff
@@ -58,6 +58,13 @@ COPY gitk.desktop /home/jovyan/Desktop/
 COPY .git_icon.png /home/jovyan/Desktop/
 RUN chmod +x /home/jovyan/Desktop/gitk.desktop
 
+#################################################################
+# Add wallpaper and force the default to point to the renku one
+
+RUN mkdir -p /usr/share/backgrouns/renku/
+COPY renku_background_dots.png /usr/share/backgrounds/renku/
+RUN ln -s -f /usr/share/backgrounds/renku/renku_background_dots.png /usr/share/backgrounds/xfce/xfce-stripes.png
```

Comment:
that's dirty! :)

Reply:
Yeah I know... 

The thing is the local `~/.config/xfce4/xfconf/xfce-perchannel-xml/xfce4-desktop.xml` profile (where we can replace the stripes.png with the renku.png) isn't generated until the VNC button is clicked.

The alternative is to do it 'cleanly' in the `post-init.sh` script, where we spin up the VNC and then in that `xfce4-desktop.xml` file do a `sed` to replace it properly. 

What do you think?

Reply:
I think this is fine - it just might be hard to find the image in case someone goes looking for it

Reply:
True. If I try to go into the desktop settings from the desktop, I get shown 

![image](https://user-images.githubusercontent.com/22785147/114569202-0e4d8480-9c75-11eb-8cf6-8b09b9f99513.png)

so there's hopefully not really any user error, unless as you say they go looking for that specific wallpaper.